NFIQ2/NFIQ2Algorithm/src/prediction/RandomForestML.cpp

```diff
@@ -150,9 +150,37 @@ NFIQ::Prediction::RandomForestML::initModule(
 
 void
 NFIQ::Prediction::RandomForestML::evaluate(
-    const std::vector<NFIQ::QualityFeatureData> &featureVector,
+    std::unordered_map<std::string, NFIQ::QualityFeatureData> &featureMap,
```

Comment:
Since we're changing it, we can figure out the type by looking to the left.

```suggestion
    std::unordered_map<std::string, NFIQ::QualityFeatureData> &features,
```